package.json

```diff
@@ -31,7 +31,8 @@
   "activationEvents": [
     "onLanguage:java",
     "onDebugInitialConfigurations",
-    "onDebugResolve:java"
+    "onDebugResolve:java",
+    "onCommand:vscJavaDebug.SpecifyProgramArgs"
```

Comment:
It is not necessary put vsc as the prefix since all the code base is working inside the VSCode scope.